src/components/global-block-link/editor.lazy.scss

```
@@ -0,0 +1,181 @@
+.spectra-global-block-style-name-modal {
+
+	> .components-modal__content {
+		width: 550px;
+		margin: 0;
+		padding: 32px;
+	}
+
+	.components-modal__header {
+		display: none !important;
+	}
+	.gbs-add-new-style-head {
```

We had a stylint rule for this, but it was disabled for some reason.

There should be an empty line before any CSS selector.
```suggestion
	}

	.gbs-add-new-style-head {
```

---

If we add space linting issue comes

---

Error should not come for new line, it will come if there is spacing in the new line.
Resolving for now though.